src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -204,11 +204,11 @@ MessagePtr TwitchMessageBuilder::build()
                 MessageColor::System, FontStyle::ChatMediumSmall)
             ->setLink({Link::ViewThread, this->thread_->rootId()});
 
-        this->emplace<TextElement>("@" + threadRoot->loginName + ":",
+        this->emplace<TextElement>("@" + threadRoot->displayName + ":",
```

Comment:
Preferably we should use the user's display name setting here

Reply:
Updated in f20538f